### PR TITLE
Issue #13213: Remove '//ok' comments from input files

### PR DESCRIPTION
**Pull request by aye-nyeinSan**
#13213

### CODE REVIEW

File: config/checkstyle-input-suppressions.xml [hunk 1]
```diff
@@ -68,52 +68,7 @@
           files="[\\/]packageobjectfactory[\\/]zoo[\\/]FooCheck.java"/>
 
   <!-- until https://github.com/checkstyle/checkstyle/issues/13213 -->
-  <suppress id="UnnecessaryOkComment"
```
Comment (MANISH-K-07) [inline config/checkstyle-input-suppressions.xml:71]: Everything lgtm....

**Review by MANISH-K-07 — COMMENTED** [review]
@aye-nyeinSan ,
It is recommended to put a comment like "I am on XXXModule" on the issue page to avoid conflicts.

The CI are all successful. All it needs is an approval from maintainers 😄

**Review by nrmancuso — APPROVED** [review]
I have approved GitHub actions workflows, good to merge as these pass.

File: config/checkstyle-input-suppressions.xml (hunk 1 above)
Comment (aye-nyeinSan) [reply config/checkstyle-input-suppressions.xml:71]: Thanks for reminding me. Later, I will comment on issue ticket @MANISH-K-07

**Comment by nrmancuso** [comment]
@aye-nyeinSan i accidentally did a merge commit instead of rebasing on master, I will fix this PR and merge it the next time I am able to get on my laptop.

**Comment by nrmancuso** [comment]
@aye-nyeinSan thanks a lot for your contribution!

**Comment by aye-nyeinSan** [comment]
Thank you @nrmancuso . I am excited to contribute to open source. I will keep watching what i can do .

**Pull request merged**